assignments/array-methods.js

```diff
@@ -56,28 +56,77 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(element){fullName.push(element.first_name + ' ' + element.last_name)});
```

Comment:
The second version is styled more preferably, though with arrow syntax you could stick this all on one line. Also, this is a good use-case for string interpolation and template literals (backticks and dollar-sign curly braces).